docs/migrating.rst

```diff
@@ -1171,6 +1171,8 @@ The following changes have been made:
 
 - :meth:`Permissions.stage_moderator` now includes the :attr:`Permissions.manage_channels` permission and the :attr:`Permissions.request_to_speak` permission is no longer included.
 
+- ``MessageType.application_command`` has been renamed to :attr:`MessageType.chat_input_command` due to the addition of context menu commands.
+
```

Comment:
```suggestion
```

Breaking changes from v2 don't get put into the migrating guide. That's for v1.7 -> v2 changes.

Reply:
Good catch, that's fixed now